src/plugins/featureFlags.ts

```diff
@@ -15,7 +15,8 @@ const defaultFlagSet: LDFlagSet = {
   'sentry-enable': false, // by default, no sentry logs
   'banner-text': '', // by default, there is no banner text
   'enable-web-chat': false, // by default, old webchat is disabled
-  'enable-genesys-web-message': false // by default, genesys web message is disabled
+  'enable-genesys-web-message': false, // by default, genesys web message is disabled
+  'is-support-society': true // by default support society
```

Comment:
This name confused me a bit until I went into LaunchDarkly and saw "namerequest support society NR in BC".

How about following the current convention and naming this `enable-society`?